Scripts/script-ADGetUser.yml

```diff
@@ -256,7 +264,6 @@ args:
   description: mail attribute of desire user
 - name: userAccountControlOut
   auto: PREDEFINED
-  description: Get translated user acount control flags
```

Comment:
Why did you remove the description?

Reply:
there were two description lines, when downloading the integration it removed the first automatically.

Reply:
👍